tests/suites/test_suite_ssl.function

```diff
@@ -3669,6 +3670,64 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS:MBEDTLS_SSL_PROTO_TLS1_3 */
+void psa_hkdf_extract( int alg, char *hex_ikm_string,
+                       char *hex_salt_string, char *hex_prk_string )
+{
+    unsigned char *ikm = NULL;
+    unsigned char *salt = NULL;
+    unsigned char *prk = NULL;
+    unsigned char *output_prk = NULL;
+    size_t ikm_len, salt_len, prk_len, output_prk_len;
+
+    output_prk_len = PSA_HASH_LENGTH( alg );
+    output_prk = mbedtls_calloc( 1, output_prk_len );
```

Comment:
Pre-existing: shoud check if the result is NULL. Actually, should use `ASSERT_ALLOC`. Can you fix the existing test as well?